dev/breeze/src/airflow_breeze/global_constants.py

```diff
@@ -188,3 +188,81 @@
 MSSQL_HOST_PORT = "21433"
 FLOWER_HOST_PORT = "25555"
 REDIS_HOST_PORT = "26379"
+
+AVAILABLE_PACKAGES = [
```

Comment:
This list should be generated from the directories available in `docs`  (minus few folders that we know we should skip: "exts", "integration-logos", "rtd-deprecation". It will change in the future and new packages will be added so it would be great to get this list automatically (it should be fast for click as well).

Reply:
@potiuk  Do i have to ignore directories starting with underscore too?

Reply:
Yes.